packages/v4/src/content/design-guidelines/demos/card-view/card-view.md

```diff
@@ -0,0 +1,81 @@
+
+<img src="./img/card-view.png" alt="card view" />
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+
+### Background colors for card views
+Use a white or a gray background for your card views.
+
+Use a white background if your content view can change. Examples:
+- A card that opens an expandable panel on the right
+- A card view that toggles into a table or list view
+
+<img src="./img/white-background.png" alt="card view with white background" /> 
+
+**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow.
+
+Use a gray background if the sole purpose of your content view is to display cards.
+
+<img src="./img/gray-background.png" alt="card view with gray background" />
+
+### Card views on mobile
+Cards will size down appropriately based on the built-in CSS responsiveness behaviors. Most of the time, they’ll follow the grid layout, expanding and compressing at certain breakpoints. At the smallest mobile size, they take up almost the entire width of the screen.
+
+<img src="./img/mobile-cards.png" alt="mobile card view" width="375"/>
+
+### Card view actions
+You can customize card actions in a few ways. Here are some recommendations to consider depending on your use case:
+
+#### Inline actions
+If an action can be performed on one card only, place the action in the card’s body or footer. You don’t need to include a checkbox because the actions are card-specific.
+
+- If you have a few cards with single actions, use the secondary button style.
+- If you have multiple cards with single actions, use a link button or link text.
+
+<img src="./img/few-multiple-actions.png" alt="few cards with multiple actions" />  
+
+Inline action cards are commonly used as navigation or drill-down indicators. In this case, present the user action on the card to differentiate this type of navigational card from a standard card. This way, users can easily distinguish the card types if your application includes both.   
+
+<img src="./img/inline-actions.png" alt="cards with inline actions" />
+
+#### Multiple inline actions
+If multiple actions can be performed on a single card, place those actions in a kebab. Position the kebab in the top-right corner of the header.
+
+<img src="./img/multiple-inline.png" alt="cards with multiple inline actions in a kebab" />
+
+#### Global actions
+If an action can be performed globally on multiple cards, or if it leads to multiple pieces of content at the same time, place the action in a toolbar above the card view. In the top-right corner of your card, add a checkbox so that users can select one or more items.
+
```

Comment:
```suggestion

To perform a global action on one or more selected cards, users can check their desired cards' checkbox. Upon hover, a card's shadow expands, and their checkbox turns blue when selected. Once a user makes their desired card selections, they can perform global actions by clicking the global action button in the toolbar.
```

This might be made clearer by writing out the sequence vs. providing a general list of behavior. At first, I thought the numbered list indicated a procedure: But if what I'm gathering is correct, we're looking to summarize expected behavior here. Above is just a suggestion for how we can accomplish communicating the action sequence and expected behavior.

Let's also move this above the example image.

Reply:
I just removed this. It's not necessary.